### DISCUSSION TITLE
Is it possible to open a GET request in a new tab?

### DISCUSSION THREAD

**Discussion by rin-yato** [scalar/scalar#1154] [General]
Let say I have an oauth endpoint, what I want is for the request to open in a new tab instead. Is that possible currently?

Thanks in advance for any help!

**Comment by hanspagel** [comment]
Not possible currently, but we just discussed this week to add it! 😁 Will take some time though.

**Reply by rin-yato** [reply]
Thanks a bunch! Looking forward to it.

Really appreciate ur effort!🔥

**Reply by marclave** [reply]
@hanspagel i have a prototype on my computer, we should chat when we meet up on tuesday ✨
